Mimir.Worker.Tests/Fixtures/States/StateReader.cs

```diff
@@ -0,0 +1,29 @@
+using Bencodex;
+using Bencodex.Types;
+using Libplanet.Common;
+
+namespace Mimir.Worker.Tests.Fixtures.States;
+
+public static class StateReader
+{
+    private static readonly string StatesFullPath = Path.GetFullPath(
+        Path.Combine(
+            Directory.GetCurrentDirectory(),
+            "../../../",
+            "Fixtures",
+            "States"));
+
+    private static readonly Codec Codec = new();
+
+    public static IValue ReadState(string fileName)
+    {
+        var file = GetFile(fileName);
+        return Codec.Decode(ByteUtil.ParseHex(file));
+    }
+
+    private static string GetFile(string fileName)
+    {
+        var path = Path.Combine(StatesFullPath, fileName);
+        return File.ReadAllText(path).Trim();
+    }
```

Comment:
I think it should be renamed to `ReadStateFile` because I feel like that it will return [`File` type](https://learn.microsoft.com/en-us/dotnet/api/system.io.file?view=net-8.0).
```suggestion
    public static IValue ReadState(string fileName)
    {
        var file = ReadStateFile(fileName);
        return Codec.Decode(ByteUtil.ParseHex(file));
    }

    private static string ReadStateFile(string fileName)
    {
        var path = Path.Combine(StatesFullPath, fileName);
        return File.ReadAllText(path).Trim();
    }
```

Reply:
Good point.